src/display/annotation_layer.js

```diff
@@ -165,6 +165,10 @@ class AnnotationElement {
     let width = data.rect[2] - data.rect[0];
     let height = data.rect[3] - data.rect[1];
 
+    if (this.data.alternativeText) {
```

Comment:
Alternative text is only defined on widget annotations, so in this file this code should also be in the `WidgetAnnotationElement` class. I would say you can put it in https://github.com/mozilla/pdf.js/blob/2f3162167d664208716013486d6697689d1d08a4/src/display/annotation_layer.js#L429-L432 since you have `this.container` there.